frontend/__mocks__/@monaco-editor/react.tsx

```
@@ -168,29 +189,38 @@ const MonacoEditor = (props: {
       onClick: () => {},
       onMouseDown: (handler) => {
         editorMockRef.current.onMouseDown = handler
+        return mockDisposable()
       },
       onDidBlurEditorWidget: (handler) => {
         editorMockRef.current.onDidBlurEditorWidget = handler
+        return mockDisposable()
       },
+      onDidFocusEditorWidget: () => mockDisposable(),
+      onDidChangeModel: () => mockDisposable(),
+      onDidChangeModelContent: () => mockDisposable(),
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)